Sdtrig.tex

```diff
@@ -319,6 +319,22 @@ \subsection{Cache Operations}
     \end{steps}
 \end{commentary}
 
+\subsection{Invalid Virtual Addresses}
+
+For virtual address matches without a mask, \RcsrTdataTwo must be able to hold
+all valid virtual addresses but it need not be capable of holding other values.
+Implementations may convert invalid virtual addresses to a different invalid
+virtual address before comparing the address to \RcsrTdataTwo.
```

Comment:
Two comments:
1. It's better to say it converts "an invalid address" (singular) to another (singular) invalid address, similar to the priv spec.  That avoids any implication that all invalid addresses must convert to one single value.
2. It should say that the value observed on tdata2 reads goes through this same transformation.

Reply:
> It should say that the value observed on tdata2 reads goes through this same transformation.

1. Can we leave tdata2 alone? This gets extremely confusing. We have several addresing modes that can change during execution: `bare`, `sv39`, etc.. How do you expect the value of `tdata2` be changed if the execution context is not constatnt?

2.As a user of the trigger I want to be able to program the trigger with whatever value I want. Moreover, the existing trigger enumeration specification imply some `tdata2` values may not be supported. That is availability of trigger already depends on `tdata2`. I would expect that no additional changes to the value written in observed.